test/unit/services/domain/test-claim-summary.js

```diff
@@ -56,6 +57,15 @@ describe('services/domain/claim-summary', function () {
     }
   })
 
+  it('should return an getNoExpensesClaimedFor validation error given no claim expenses', function () {
+    try {
+      claimSummary = new ClaimSummary(VALID_VISIT_CONFIRMATION, VALID_BENEFIT_UPLOAD_NEEDED, VALID_BENEFIT_DOCUMENT, [], VALID_IS_ADVANCE_CLAIM, BENEFIT_UPLOAD_NOT_REQUIRED)
+    } catch (e) {
+      expect(e).to.be.instanceof(ValidationError)
+      expect(e.validationErrors['claim-expense'][0]).to.equal(ERROR_MESSAGES.getNoExpensesClaimedFor())
+    }
+  })
```

Comment:
This comment really applies to this whole test suite. But there is no need to wrap the exception throwing function in a try catch. Chai handles this for you. The above can be rewritten as: 

```
return expect(function() {
    new ClaimSummary(VALID_VISIT_CONFIRMATION, VALID_BENEFIT_UPLOAD_NEEDED, VALID_BENEFIT_DOCUMENT, [], VALID_IS_ADVANCE_CLAIM, BENEFIT_UPLOAD_NOT_REQUIRED)
  }).to.throw(ValidationError)
```

As the number of parameters means the new ClaimSummary line is trailing off the page I would also suggest putting new lines after each parameter.